src/agent/inspectordebugapi.ts

```diff
@@ -46,9 +46,17 @@ export class InspectorDebugApi implements debugapi.DebugApi {
   breakpoints: {[id: string]: BreakpointData} = {};
   sourcemapper: SourceMapper;
   session: inspector.Session;
-  scriptmapper: {[id: string]: any} = {};
-
+  // TODO: listeners, scrpitmapper, location mapper and breakpointmapper can use
+  // Map in the future after resolving Map.prototype.get(key) returns V or
+  // undefined.
   listeners: {[id: string]: utils.Listener} = {};
+  // scriptmapper maps scriptId to actual script path.
+  scriptmapper: {[id: string]: any} = {};
```

Comment:
Use camelCase, for scriptmapper, locationmapper, and breakpointmapper

Reply:
Done.